virt/virt-web-console-overview.adoc

```diff
@@ -441,7 +441,7 @@ The *Overview* tab displays resource usage, alerts, and configuration informatio
 |General VirtualMachine information
 
 |"Utilization" tile
-|*CPU*, *Memory*, *Storage*, and *Network transfer* charts
+|*CPU*, *Memory*, *Storage*, and *Network transfer* charts. By default, *Network transfer* displays the sum of all networks. To view the breakdown for a specific network, click *Breakdown by network*.
```

Comment:
The use of `"` for these tile names is interesting. Do you happen to have a reference for this handy? I seldom have to describe GUI items, and I would def have missed this convention if describing a tile :nerd_face:

Reply:
Tagging @apinnick since she built this assembly and I'm not familiar with the `""` specifics.

Reply:
@jeana-redhat @sabrinajess 
Our beloved [ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tile) came up with that distinction (** vs ""). Makes no sense to me but who am I to argue with Big Blue?

![image](https://user-images.githubusercontent.com/31725355/228817255-ff946a65-a1d9-4f39-9b16-39a53aa875dd.png)

Reply:
Thanks, Avital! It is a little funny to me too, hopefully funny enough that I remember it when needed :sweat_smile: